Cargo.toml

```diff
@@ -1,5 +1,5 @@
 [workspace.package]
-version = "0.2.1"
+version = "0.2.2"
```

Comment:
This will trigger a new release. I don't want to do this until our release CI is fixed. 

[This](https://github.com/anza-xyz/agave/pull/4642) agave PR should be merged soon and make it easy to fix CI (hopefully). If they don't release soon I can try other approaches

Reply:
I've managed to get the build working, so this is resolved!